Communitter/src/main/java/com/communitter/api/controller/PostController.java

```diff
@@ -29,4 +33,14 @@ public ResponseEntity<String> deletePost(@PathVariable Long id){
         return ResponseEntity.ok(postService.deletePost(id));
     }
 
+    @PreAuthorize("@authorizer.checkSubscription(#root,#id)")
+    @PostMapping("/vote/{id}")
+    public ResponseEntity<PostVote> votePost(@P("id") @PathVariable Long id, @RequestParam  boolean isUpvote){
+        return ResponseEntity.ok(postService.votePost(id, isUpvote));
+    }
+
+    @GetMapping("/voteCount/{id}")
```

Comment:
{id} vote id gibi algılanıyor. posts/{id}/vote-count daha uygun olabilir.

Reply:
Thanks for the hint. It is fixed.